Tests/SPMTest/CoreSPMTest/CoreSPMTest/ContentView.swift

```diff
@@ -0,0 +1,29 @@
+import SwiftUI
+import MapboxCoreNavigation
+import MapboxDirections
+import CoreLocation
+
+struct ContentView: View {
+    var body: some View {
+        Button("Request route") {
+            let routeOptions = RouteOptions(coordinates: [
+                CLLocationCoordinate2D(latitude: 10, longitude: 10),
+                CLLocationCoordinate2D(latitude: 11, longitude: 11)
+            ])
+            Directions.shared.calculate(routeOptions) { _, result in
```

Comment:
Could you use `calculateWithCache` to check `MapboxCoreNavigation` is imported?

Reply:
I would not advise to use exactly `calculateWithCache` as this method most likely will be [removed](https://github.com/mapbox/mapbox-navigation-ios/pull/3261/files#diff-c3b17629de0ab1de692fbf26006d4edeb545b1f47dbd97b63ec55ba686d2b6a4L19) when we implement `HybridRouter`.

Reply:
`RouteOptions` is already a part of `MapboxCoreNavigation`

Reply:
Sorry, Xcode is lying, `RouteOptions` is a part of `Directions`
<img width="383" alt="image" src="https://user-images.githubusercontent.com/1976216/133994969-bfbc9b12-a315-422b-a27c-669518da1878.png">

Reply:
I added `NavigationEventsManager` usage, so now we can be sure that MapboxCoreNavigation is linked.